net-voip/sipvicious/sipvicious-0.3.4.ebuild

```diff
@@ -0,0 +1,30 @@
+# Copyright 2020-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{7..10} )
+DISTUTILS_USE_SETUPTOOLS=rdepend
```

Comment:
Can remove this
```
 * QA Notice: DISTUTILS_USE_SETUPTOOLS value is probably incorrect
 *   have:     DISTUTILS_USE_SETUPTOOLS=rdepend
 *   expected: (unset)
```

Reply:
Any update on addressing some of the other points I mentioned like this one? (there are others I haven't heard back from)

There's even a bug opened for this now
https://bugs.gentoo.org/809473